test/rand.jl

```diff
@@ -22,4 +33,11 @@ using StaticArrays
         @test rand(Y) isa SVector{2,T}
     end
 
+    for T in (Float32, Float64, BigFloat)
+        X = Interval{T}(3, 7)
+        Y = rand(X,7)
```

Comment:
Maybe you can test that the type of `Y` is correct too.
Also, please add a space after the comma.